File: src/targets/csharp/httpclient/fixtures/cookies.cs
var clientHandler = new HttpClientHandler
{
    UseCookies = false,
};
var client = new HttpClient(clientHandler);
var request = new HttpRequestMessage
{
    Method = HttpMethod.Post,
    RequestUri = new Uri("http://mockbin.com/har"),
    Headers =
    {
        { "cookie", "foo=bar; bar=baz" },
    },
};
using (var response = await client.SendAsync(request))
{
    response.EnsureSuccessStatusCode();
    var body = await response.Content.ReadAsStringAsync();
    Console.WriteLine(body);
}

File: src/targets/dart/http/fixtures/cookies.dart
import 'package:http/http.dart' as http;

void main() async {

  final headers = {
    'cookie': 'foo=bar; bar=baz',
  };

  final response = await http.post(
    Uri.parse('http://mockbin.com/har'),
    headers: headers,
  );

  print(response.statusCode);
  print(response.body);

}